content/en/docs/contrib/new-contributors.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Advice for new contributors
+description: Are you a contributor and not sure what to do? Want to help but just don't know how to get started? This is the section for you.
+weight: 1
+aliases:
+- /docs/contrib/feedback
+- /docs/spin-operator/support/feedback
+---
+
+This page contains more general advice on ways you can contribute to SpinKube, and how to approach
+that.
+
+If you are looking for a reference on the details of making code contributions, see the [Writing
+code]({{< ref "writing-code" >}}) documentation.
+
+## First steps
+
+Start with these steps to be successful as a contributor to SpinKube.
+
+1. Join the conversation
+
+It can be argued that collaboration and communication are the most crucial aspects of open source
```

Comment:
Nit: can these sub-sections be indented 4 spaces so that they appear nested under each numbered section?